### DISCUSSION TITLE
Calling a Webhook on Button Click and Displaying Response

### DISCUSSION THREAD

**Discussion by Yakito** [Budibase/budibase#10913] [Help] [answered]
I'm currently working on a feature and I'm not sure if it's possible to implement it with Budibase or not. I'd like to call a webhook when a button is clicked, and then display the response of that webhook back to the user.

The desired workflow is as follows:

User clicks a button in the Budibase app.
This button click triggers a webhook (POST request to a specified URL).
The server processes this request and sends back a JSON response.
This response is displayed back to the user in the Budibase app.
I've looked through the documentation and the community posts, but I couldn't find a clear answer on whether this is possible or not.

Is this something that can be achieved using Budibase? 
Thanks a lot!

**Comment by melohagan** [comment, marked as answer]
Hey @Yakito 

This is possible with the new **Collect Data** automation action!

More here: https://docs.budibase.com/docs/synchronous-automations
